package.json

```diff
@@ -11,7 +11,7 @@
     "lodash": "^4.17.21",
     "react": "^18.2.0",
     "react-redux": "^8.1.3",
-    "terraso-backend": "github:techmatters/terraso-backend#3d323b5",
+    "terraso-backend": "github:techmatters/terraso-backend#7f438224bb5997cd6a13b3d0fbbaf8ef3f45f3d3",
```

Comment:
issue: please use a short hash.

Reply:
This will be changed when we get the final hash after the backedn PR is merged